exotations/solvers/exotica_ddp_solver/src/ddp_solver_py.cpp

```diff
@@ -0,0 +1,52 @@
+//
+// Copyright (c) 2018, University of Edinburgh
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//
+//  * Redistributions of source code must retain the above copyright notice,
+//    this list of conditions and the following disclaimer.
+//  * Redistributions in binary form must reproduce the above copyright
+//    notice, this list of conditions and the following disclaimer in the
+//    documentation and/or other materials provided with the distribution.
+//  * Neither the name of  nor the names of its contributors may be used to
+//    endorse or promote products derived from this software without specific
+//    prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+// LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+// POSSIBILITY OF SUCH DAMAGE.
+//
+
+#include <exotica_core/exotica_core.h>
+#undef NDEBUG
+#include <pybind11/eigen.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <exotica_ddp_solver/analytic_ddp_solver.h>
+#include <exotica_ddp_solver/control_limited_ddp_solver.h>
+
+using namespace exotica;
+namespace py = pybind11;
+
+PYBIND11_MODULE(exotica_ddp_solver_py, module)
+{
+    module.doc() = "Exotica DDP Solver";
+
+    py::module::import("pyexotica");
+
+    py::class_<AnalyticDDPSolver, std::shared_ptr<AnalyticDDPSolver>, MotionSolver> analytic_ddp_solver(module, "AnalyticDDPSolver");
```

Comment:
I think you should be able to expose FeedbackMotionSolver in the normal pyexotica, and then get_feedback_control as part of it. Then when you expose AnalyticDDPSolver, you need to declare it as FeedbackMotionSolver. This should work - so you only need to include the correct module and that's it.